Dockerfile

```diff
@@ -18,25 +25,14 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone &
     clang-tidy-14 \
     clang-format-14 \
     libpng-dev && \
-    apt clean && \
+    practicerom-dev && \
+    apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-# Install practicerom
-RUN curl https://practicerom.com/public/packages/debian/pgp.pub | \
-    apt-key add - && echo deb http://practicerom.com/public/packages/debian staging main >/etc/apt/sources.list.d/practicerom.list && apt update
-
-RUN apt-get install -y practicerom-dev
-
-COPY requirements.txt requirements.txt
-
-RUN python3 -m pip install --no-cache-dir -r requirements.txt
-
-ENV LANG C.UTF-8
-
-RUN mkdir /mm
-
 WORKDIR /mm
-
 RUN git config --global --add safe.directory /mm
 
-ENTRYPOINT ["/bin/bash", "-c"]
+CMD ["/bin/sh", "-c", \
+    "make -j $(nproc) init && \
+    echo Completed build. Open another terminal and run 'docker-compose exec mm sh' to shell into this running container. && \
+    tail -f /dev/null"]
```

Comment:
I don't think setting up the docker image should build the rom too. Opening an interactive terminal would be better and more in line with the the workflow of the other build/usage methods

Reply:
I actually did not intend to commit that change, that was something I was working on locally. Accidentally forgot to change it back to address your feedback :D